build.cmd

```diff
@@ -1 +1 @@
-msbuild /r %~dp0LibraryManager.sln /clp:Verbosity=Minimal;Summary;ForceNoAlign %*
+msbuild /r %~dp0LibraryManager.sln /clp:Verbosity=Minimal;Summary;ForceNoAlign /bl:%~dp0artifacts/Build.binlog %*
```

Comment:
How much time does capturing binlog add? Do we want to do this on every build, even if everything succeeded?

Reply:
It's a negligible overhead, much less than having the console output turned up.  Yes, it's nice to collect it on good builds, I do that a lot when diffing a build break (before vs. after), or validating that specific steps were carried out correctly (signing, loc, etc).